MinGL.cpp

```diff
@@ -169,6 +170,103 @@ void MinGL::putPixel(int x, int y, const MinGLColor& color, int width, int heigh
 	glDisable(GL_SCISSOR_TEST);
 }
 
+const MinGLColor MinGL::getPixelColor(int x, int y, int width, int height) const{
```

Comment:
Not used in this function. Please remove it.

Reply:
thanks for the review, I've removed `width` and `height` parameters, also removed the redundant `#include<queue>`